Ryujinx.Audio.Backends.SDL2/SDL2HardwareDeviceDriver.cs

```diff
@@ -112,13 +113,19 @@ internal static uint OpenStream(SampleFormat requestedSampleFormat, uint request
 
             if (device == 0)
             {
+                string errorMessage = $"SDL2 open audio device initialization failed with error \"{SDL_GetError()}\"";
+
+                Logger.Error?.Print(LogClass.Application, errorMessage);
```

Comment:
```suggestion
                Logger.Error?.Print(LogClass.Application, 
                    $"SDL2 open audio device initialization failed with error \"{SDL_GetError()}\"");
```
Add a new extra empty line before the `return` too